aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -62,6 +69,13 @@ Tensor to(
            "to(options) expects unset requires_grad flag, but got "
            "options.requires_grad set as ", options.requires_grad());
 
+  if (options.device().type() == DeviceType::Vulkan
```

Comment:
why do you need it here duplicated? it's already in to_impl

Reply:
Removing it in https://github.com/pytorch/pytorch/pull/39019